src/sentry/models/options/project_template_option.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model, sane_repr
+from sentry.db.models.fields import PickledObjectField
+
+
+@region_silo_only_model
```

Comment:
```suggestion
@region_silo_model
```

This is in the process of being renamed.